apps/greenhouse/src/hooks/useApi.js

```diff
@@ -47,7 +47,9 @@ const useApi = () => {
           // temporary fix to forward initialFilters to the Plugins until middleware is implemented
           const appProps = { username: authData?.parsed?.fullName }
           appProps.initialFilters = {
-            support_group: authData?.parsed?.supportGroups?.map((group) => group),
+            ...(Array.isArray(authData?.parsed?.supportGroups) && {
```

Comment:
Could you move the if statement outside of this object? If Array.isArray returns false, the entire statement will evaluate to false. I’m not sure what the result will be in that case.

Reply:
already checked. It doesn't include any object if supportGroups doesn't exist or is not an array

Reply:
This is the shortest prettier way i found :)

Reply:
It can be tested very easy in the console:
![Screenshot 2024-11-14 at 13 47 41](https://github.com/user-attachments/assets/0b216c85-663a-4839-8cc0-919fe166c512)